upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -234,6 +239,34 @@ func (c *ApplyClusterCmd) Run(ctx context.Context) error {
 
 	cluster := c.Cluster
 
+	configBase, err := vfs.Context.BuildVfsPath(cluster.Spec.ConfigBase)
+	if err != nil {
+		return fmt.Errorf("error parsing config base %q: %v", cluster.Spec.ConfigBase, err)
+	}
+
+	if !c.AllowKopsDowngrade {
+		kopsVersionApplied, err := configBase.Join(registry.PathKopsVersionApplied).ReadFile()
+		if err == nil {
+			version, err := semver.Parse(strings.TrimSpace(string(kopsVersionApplied)))
+			if err != nil {
+				return fmt.Errorf("error parsing last kops version applied: %v", err)
```

Comment:
```suggestion
				return fmt.Errorf("error parsing last kops version used: %v", err)
```

Reply:
It's not the last version used. One can use a downrev or uprev kops to edit the cluster or ig specs; it only matters which version of kops last did an `apply_cluster`.

Reply:
I understand this, but it's more from the developer's point of view. "Used" is more user oriented.
In any case, even using "last kops version used to apply changes" would be a bit better.

Reply:
Perhaps "updated", since the CLI command is `kops update cluster --yes`. The `apply_cluster.go` is mixing terminology a bit.

Reply:
That would be even better.

Reply:
Went with the "updated" terminology.

Reply:
Probably you want to update this message also.